modules/virt-about-qemu-guest-agent-web.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-viewing-qemu-guest-agent-information.adoc
+
+[id="About-the-qemu-guest-agent-web_{context}"]
+= About the QEMU guest agent information in the web console
+
+When the QEMU guest agent is installed, the *Details* pane within the *Virtual machine overview* tab and the *Details* tab display information about the *Hostname*, *Operating system*, *Time zone*, and *Logged in users*.
+
+The *Virtual machine overview* shows information about the guest operating system installed on the virtual machine.
+
+The *Details* tab displays a table for *Logged in users.* This information includes *User name*, *Domain*, *Log in time*, and *Elapsed log in time*.
```

Comment:
It's "Login time" and "Elapsed logged in time".

Reply:
Contacted @pcbailey with proposed changes to these terms.

Reply:
`Log in time` should be `Login time`.
Others look good to me.